app/server/server.ts

```diff
@@ -49,8 +49,5 @@ server.use(routes.productsProvider("/api/"));
 // ALL OTHER ENDPOINTS CAN BE HANDLED BY CLIENT SIDE REACT ROUTING
 server.use(routes.frontend);
 
-if (conf.SERVER_DSN) {
-  server.use(Raven.errorHandler());
-}
```

Comment:
Is this part implicit in the new Sentry version?

Reply:
I thought it was, because when I tested and faked some errors it seemed to catch them, however according to https://sentry.io/for/express/ I should still include them, will make some changes accordingly.

Reply:
More detail : https://docs.sentry.io/platforms/node/express/

Reply:
done